### PR TITLE
Add 'EmmetOneline' Package

### PR DESCRIPTION
**Pull request by kakRostropovich**

### CODE REVIEW

File: repository/e.json [hunk 1]
```diff
@@ -393,7 +393,7 @@
 		{
 			"name": "Emmet Css Snippets",
 			"details": "https://github.com/P233/Emmet-Css-Snippets-for-Sublime-Text-2",
-			"labels": ["snippets"],
+			"labels": ["css"],
```
Comment (FichteFoll) [inline repository/e.json]: Please don't change labels of other packages.

**Comment by FichteFoll** [comment]
You have to push a valid [semantic version](http://semver.org) [tag](http://git-scm.com/book/en/v2/Git-Basics-Tagging) (with optional leading `v`) to your repo.

File: repository/e.json (hunk 1 above)
Comment (kakRostropovich) [inline repository/e.json]: Fixed it in last commit

**Comment by kakRostropovich** [comment]
I added a version in the correct form in README.md. I understood everything correctly?

**Comment by FichteFoll** [comment]
The form is correct, but the version needs to appear here: https://github.com/kakRostropovich/sublime-emmet-css-from-one-line/tags

**Comment by kakRostropovich** [comment]
Added release in project, look here please

**Comment by kakRostropovich** [comment]
Thank you, but in my plugin page (https://packagecontrol.io/packages/EmmetOneLine) not visible gif image. Did I add it wrong?

**Comment by FichteFoll** [comment]
It is for me. Maybe it was caching still.

**Comment by kakRostropovich** [comment]
Now all is well, thank you!

**Pull request merged**